### PR TITLE
AC Automated Test - LRAC-9865, LRAC-10003,LRAC-10004,LRAC-10005 and LRAC-10006 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-9865
https://issues.liferay.com/browse/LRAC-10003
https://issues.liferay.com/browse/LRAC-10004
https://issues.liferay.com/browse/LRAC-10005
https://issues.liferay.com/browse/LRAC-10006
https://issues.liferay.com/browse/LRAC-9883 (Bug ticket)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsSettings.testcase [hunk 1]
```diff
@@ -0,0 +1,513 @@
...
+
+			ACSettings.goToDefinitions();
+
+			ACSettings.goToEvents();
+
+			ACNavigation.openItem(itemName = "Custom Events");
+		}
+
+		task ("Edit the new custom events") {
+			ACNavigation.openItem(itemName = "${customEventName}");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsSettings.testcase:115]: Hi @ViniLopes20 , please put the `events` word in the singular.

`task ("Edit the new custom event")`

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
It's covered by https://github.com/liferay-qa/liferay-portal/pull/1355

**Pull request closed without merging**